core/common/src/main/java/alluxio/wire/ThriftUtils.java

```diff
@@ -220,5 +247,30 @@ public static WorkerNetAddress fromThrift(alluxio.thrift.WorkerNetAddress worker
   public static alluxio.thrift.WorkerNetAddress toThrift(WorkerNetAddress workerNetAddress) {
     return workerNetAddress.toThrift();
   }
+
+  /**
+   * Converts wire type to thrift type.
+   *
+   * @param ttlAction {@link TtlAction}
+   * @return {@link TTtlAction} equivalent
+   */
+  public static TTtlAction toThrift(TtlAction ttlAction) {
```

Comment:
Could we combine the ttl time and ttl action into one ttl object? It seems like these two concepts should be grouped together. This should also allow you to put the conversion info in a consolidated class instead of utils.